assets/parameters.json

```diff
@@ -0,0 +1,6 @@
+{
+  "input": "/home/ubuntu/environment/github/phylorthology/assets/samplesheet.csv",
+  "fasta_dir": "/home/ubuntu/environment/github/phylorthology/assets/nf-test/full-data/",
+  "test_fasta_dir": "/home/ubuntu/environment/github/phylorthology/assets/nf-test/test-data/",
```

Comment:
Yeah, these are custom (except for input) user-specified params. Most could also be specified in the commandline, but there is (annoyingly) no way to specify several inflations parameter for be handled with the "each" operator in nextflow as needed when testing MCL inflation params. 

As for relative paths, I honestly am not sure - others can chip in here. I think that if we're pulling data/samplesheets down from s3, these could be relative, as in upstream of the working directory. I just have an aversion to relative paths in nextflow ever since it took me a day (probably two) to figure out that you couldn't use a tilda or variable in filepaths.

Reply:
I don't know if relative paths work when passing a parameters file or samplesheet, it works when you provide for example `--input ../../path/blah/blah/"*_{1,2}.fq.gz"` for running on all PE sequences in a directory. On S3 you need to give the full bucket path